main.cpp

```diff
@@ -107,6 +128,76 @@ Thing 4) House Plant
     3) extends it’s overall height per day
  */
 
+/*
+Thing 5) Waveform display
+5 properties:
+    1) Length of sample in ms 
+    2) Height in pixels 
+    3) Width in pixels
+    4) Zoom function as a percentage
+    5) Loop hightlight colour
+3 things it can do:
+    1) display transient positions of digital audio signal 
+    2) display playback head position
+    3) display user defined loops points of sample
+ */
+
+/*
+Thing 6) Control panel display
+5 properties:
+    1) Height in pixels
+    2) Width in pixels 
+    3) Number of parameters
+    4) Parameter knob size in pixels
+    5) Button size in pixels
+3 things it can do:
```

Comment:
A lot of these properties are common to a Display class.  Add some new things specific to each of these subobjects and aren't shared between them. 
- [ ] revise